tests/unit/helpers.js

```diff
@@ -1315,3 +1315,9 @@ export const createFakeBlockResult = ({
     ...others,
   };
 };
+
+export const fakesponsoredShelf = Object.freeze({
+  addons: Array(6).fill(fakeAddon),
```

Comment:
Isn't it a problem that all 6 add-ons have the exact same data (ids)?

Reply:
I don't think we have `click` URLs in those objects, though

Reply:
True. I guess I can add that in here.